content/07.pathogenesis.md

```diff
@@ -79,3 +79,17 @@ There is a high level of sequence homology between SARS-CoV-2 and SARS-CoV, and
 However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, prohibiting effective cross-neutralization across different SARS strains [@doi:10.1101/2020.03.21.990770].
 Consequently, earlier proteome-level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074].
 Considering the paucity of omics-level big data sets for SARS-CoV-2 up until now, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.
+Using such databases, the systems level reconstruction of the PPI (Protein-Protein Interaction) enabled the generation of hypotheses on the mechanism of action of SARS-CoV-2 and suggested drug targets.
+
+#### Protein-protein interaction networks
+
+In a first study [@doi:10.1101/2020.03.22.002386], 26 of the 29 SARS-CoV-2 proteins were cloned and expressed in HEK293T kidney cells and 332 high confidence human proteins that interact with them were consequently identified. 
+Notably, this study suggested the interaction of SARS-CoV-2 with innate immuty pathways, with a similarity to Mycobacterium Tuberculosis host-pathogen interactome.
+In addition, it was suggested that the envelope protein E could disrupt the host bromodomain-containing protein BRD binding to histone.
+The Spike protein S could likely intervene in the virus fusion through modualing the GOLGA7-ZDHHC5 acyl-transferase complex to increase palmitoylation.
+
+Another study [@doi:10.1101/2020.03.31.019216], used patient-derived Peripheral Blood Mononuclear cells (PBMCs) to identify 251 host proteins directly targeted by SARS-CoV-2 and the disruption of 200 host proteins following the infection.
```

Comment:
Peripheral Blood Mononuclear cells (PBMCs) -> peripheral blood mononuclear cells (PBMCs)

Reply:
fixed in #660b45f